docs/install/create-project.md

```diff
@@ -62,6 +62,7 @@ To avoid RAM and disk swap limitations, we recommend either temporarily resizing
 -   {term}`Node.js` LTS 20.x
 -   {term}`Yeoman`
 -   {term}`Yarn`
+-   {term}`git`
```

Comment:
If we make `git` a requirement, then we also need to provide a new section and reference to how to install it. We can use a tabbed interface for various OSes using `sphinx-design`, or just give a link to it. I'm more inclined to just a link because developers have probably already installed git at some point in their lives.

Reply:
Done in https://github.com/plone/volto/pull/5769 and [07f76da](https://github.com/plone/documentation/pull/1628/commits/07f76dab395be4c561f5d73b225e8fc3caca56c6)